A/AWSCRT/build_tarballs.jl

```diff
@@ -42,7 +42,7 @@ dependencies = [
     Dependency("aws_c_http_jll"; compat="0.7.12"),
     Dependency("aws_c_iot_jll"; compat="0.1.17"),
     Dependency("aws_c_mqtt_jll"; compat="0.8.12"),
-    Dependency("aws_c_s3_jll"; compat="0.3.17"),
+    Dependency("aws_c_s3_jll"; compat="0.8.0"),
```

Comment:
@quinnj these PRs are all stuck because you're updating too much :melting_face:

Reply:
Working on a fix